### PR TITLE
docs(faq): new faq account page

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Created a new FAQ page for Account, as part of the content that will be necessary to launch the console Help Center.

### CODE REVIEW

File: faq/account.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+content:
+  h1: Account
+hero: assets/general.webp
+---
+
+<Concept>
+
+## My account is locked. What do I do?
+
+If your account is locked, contact our support team to learn the reason for the blockage and discover how to regain access.
```
Comment (RoRoJ) [inline faq/account.mdx:14]: Maybe add an email address or info of how to contact them? I guess if the account is locked they can't log in to the console to open a ticket?

File: faq/account.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+</Concept>
+
+<Concept>
+
+## Can I change my Organization’s country?
+
+If you need to change your Organization country and region, you must [create a support ticket](https://console.scaleway.com/support/tickets/create).
+
+To make the process faster, when creating your ticket, inform us your Organization’s new address, including: street address, city, postal code, region, and country.
```
Comment (RoRoJ) [inline faq/account.mdx:24]:
```suggestion
To make the process faster, when creating your ticket, inform us of your Organization’s new address, including: street address, city, postal code, region, and country.
```

File: faq/account.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+You can get more quotas by adding and verifying your [payment method](/console/my-account/how-to/manage-billing-information/) and [identity](/console/my-account/how-to/verify-identity/).
+
+If you have already completed these two steps and still need an increase in quotas, contact our [support team](https://console.scaleway.com/support/tickets/create) and inform them which product you need more quotas for.
+
+</Concept>
+
+<Concept>
+
+## I don’t seem to have permission to my Organization's resources
```
Comment (RoRoJ) [inline faq/account.mdx:72]:
```suggestion
## I don’t seem to have access to my Organization's resources
```

File: faq/account.mdx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+On the other hand, when you delete an account, we permanently erase the existing resources and any data and settings. It is not possible to recover a deleted account.
+ 
+</Concept>
+
+<Concept>
+
+## How do I deactivate my Organization?
+
+Deactivating an Organization **permanently deletes** all your services and backups.
```
Comment (RoRoJ) [inline faq/account.mdx:96]: This seems to go against what was previously said - that when accounts were deactivated data and settings and resources are preserved?

"If you only want to pause your account and keep your existing resources, you can [deactivate it](/console/account/how-to/close-account/#how-to-deactivate-your-organization) instead."

"When your account is deactivated, we preserve your data and settings, making it possible to recover it later if you wish."

File: faq/account.mdx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+</Concept>
+
+<Concept>
+
+## I have created a resource in the wrong Project. Can I move it?
+
+At this moment, it is not possible to move a resource from one Project to another.
+
+If you need to move a resource, you must delete it and create it again on the right Project.
```
Comment (RoRoJ) [inline faq/account.mdx:136]:
```suggestion
If you need to move a resource, you must delete it and create it again in the right Project.
```

File: faq/account.mdx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+</Concept>
+
+<Concept>
+
+## Where can I find my Organization ID?
+
+You can find your Organization ID by accessing the [Settings](https://console.scaleway.com/organization/settings) page of your Organization dashboard.
+
+On the first section, **Organization Information**, you will find the ID and a shortcut to copy it, if necessary.
```
Comment (RoRoJ) [inline faq/account.mdx:146]:
```suggestion
In the first section, **Organization Information**, you will find the ID and a shortcut to copy it, if necessary.
```

File: faq/account.mdx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+</Concept>
+
+<Concept>
+
+## I've been invited to an Organization, but can’t see any resource
```
Comment (RoRoJ) [inline faq/account.mdx:152]:
```suggestion
## I've been invited to an Organization, but can’t see any resources
```

File: faq/account.mdx [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+When you create a Scaleway account after being invited to join a specific Organization, a personal Organization is automatically created for yourself.
```
Comment (RoRoJ) [inline faq/account.mdx:154]:
```suggestion
When you create a Scaleway account after being invited to join a specific Organization, a personal Organization is automatically created for you.
```

File: faq/account.mdx (hunk 4 above)
Comment (jcirinosclwy) [reply faq/account.mdx:96]: Thank you, Ro. You're right. 
I'm trying to confirm the precise conditions with Excellence to update the text. 🙏🏾

File: faq/account.mdx [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+
+## How do I change my email address?
+
+To change your email address:
+
+1. On the [Scaleway console](https://console.scaleway.com), go to your [profile](https://console.scaleway.com/account/profile).
+2. Under the section **User information**, click **Edit profile**.
+3. Enter a new address under **Email address**, then click **Confirm changes** to save your modification.
+
+Your email address is now updated.
```
Comment (jsudraud) [inline faq/account.mdx:56]:
```suggestion
Confirm your new email address by clicking the link sent to your inbox. Your email address is now updated.
```

**Review by jsudraud — APPROVED** [review]

File: faq/account.mdx [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+
+<Concept>
+
+## How do I change my email address?
+
+To change your email address:
+
+1. On the [Scaleway console](https://console.scaleway.com), go to your [profile](https://console.scaleway.com/account/profile).
+2. Under the section **User information**, click **Edit profile**.
+3. Enter a new address under **Email address**, then click **Confirm changes** to save your modification.
```
Comment (jcirinosclwy) [inline faq/account.mdx:54]:
```suggestion
3. Enter a new address under **Email address**, then click **Confirm changes** to save your modification.
4. Confirm your new email address by clicking the link sent to your inbox.
```

File: faq/account.mdx (hunk 9 above)
Comment (jcirinosclwy) [reply faq/account.mdx:56]: Thanks! Added the first part as a 4th step. :)

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**